src/App.js

```diff
@@ -1,5 +1,51 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { randomNum } from './utils/randomNum.js'
+import { userInput } from './utils/userInput.js'
+import { restartGame } from './utils/restartGame.js'
+import { resultStrikeBall } from './utils/resultStrikeBall.js'
+import { message } from './const/message.js'
```

Comment:
```suggestion
import { MESSAGE } from './constants/message.js'
```
제 개인적은 생각은 폴더는 `const`로 축약하지 않고 `constants`로 적어도 좋을 것 같아요~! 
그리고 상수 개념의 메세지인 것 같아서 `message`도 스네이크 케이스로 `MESSAGE`로 적는 것도 괜찮을 것 같아요!

Reply:
맞습니다! 상수 개념이다보니 constants라고 적어두는게 더 직관적으로 이해하기 쉽겠네요!
message또한 MESSAGE로 변경해야겠네요! 저것만 저렇게 작성했군요 ㅎㅎ ;;